.travis.yml

```diff
@@ -44,9 +44,6 @@ matrix:
     - os: linux
       language: python
       env: ZSH_BUILD_VERSION=zsh-4.3.11
-    - os: linux
-      language: python
-      env: ZSH_BUILD_VERSION=zsh-4.3.5
```

Comment:
If we remove this, would we have caught 483a7ad256fc6ad4bd36a990df2dc1048b995998?